aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/util/AuthProviderConverter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.cognito.util;
+
+import com.amplifyframework.auth.AuthProvider;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Helpful util class to convert AuthProvider values to Cognito Identity Provider values.
+ */
+public final class AuthProviderConverter {
+    private static final Map<AuthProvider, String> CONVERT_AUTH_PROVIDER;
+
+    /**
+     * Dis-allows instantiation of this class.
+     */
+    private AuthProviderConverter() { }
```

Comment:
Javadoc is unnecessary :)

Also, we no longer throw an error for instantiating with 
```
private AuthProviderConverter() {}
```
where there's no space between curly brackets